app/views/shared/_single_page_notification_button.html.erb

```diff
@@ -21,4 +21,4 @@
   margin_bottom: 6,
   button_location: "top",
   skip_account: skip_account,
-} if @content_item.has_single_page_notifications? %>
+} if @content_item.display_single_page_notification_button? %>
```

Comment:
I suppose you could put the whole contents of this file inside an `if` instead of using the `if` here, as the lines 1-15 don't need to be executed and I suppose it would make it a bit more readable, as it would be clear from the beginning that those things only need to be executed under the condition?

```ruby
<% if @content_item.display_single_page_notification_button? %>
...
<% end %>
```